### PR TITLE
Added edo font in title

### PR DESCRIPTION
**Pull request by bhavy2202**
Solved issue #17

### CODE REVIEW

**Comment by bhavy2202** [comment]
image

File: assets/css/styles.css [hunk 1]
```diff
@@ -1,10 +1,17 @@
+@import url(//db.onlinewebfonts.com/c/d2c52a64cb362f280d891dd4980160e5?family=Edo);
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:1]: No need of adding these codes

/* Edo Font */

@import url('http://fonts.cdnfonts.com/css/edo');

Just add this code only @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -32,6 +32,8 @@
         integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous">
     <link rel="stylesheet"
         href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css">
+    <link rel="stylesheet"
```
Comment (YoshithaRathnayake) [inline index.html:35]: remove this also @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by bhavy2202** [comment]
okay, on it.

File: index.html [hunk 3]
```diff
@@ -32,6 +32,7 @@
         integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous">
     <link rel="stylesheet"
         href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css">
+
```
Comment (YoshithaRathnayake) [inline index.html:35]: Can you remove this additional line @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by bhavy2202** [comment]
Done, @YoshithaRathnayake

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @bhavy2202

**Pull request merged**